src/TogglePassword/doc/index.rst

```diff
@@ -0,0 +1,207 @@
+Symfony UX TogglePassword
+=========================
+
+Symfony UX TogglePassword is a Symfony bundle providing visibility toggle for password inputs
+in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+It allows visitors to switch the type of password field to text and vice versa.
+
+Installation
+------------
+
+Before you start, make sure you have `StimulusBundle configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-toggle-password
+
+If you're using WebpackEncore, install your assets and restart Encore (not
+needed if you're using AssetMapper):
+
+.. code-block:: terminal
+
+    $ npm install --force
+    $ npm run watch
+
+    # or use yarn
+    $ yarn install --force
+    $ yarn watch
+
+Usage
+-----
+
+Any ``PasswordType`` can be transformed into a toggle password field by adding the ``toggle`` option::
+
+    // ...
+    use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+
+    class CredentialFormType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
+        {
+            $builder
+                // ...
+                ->add('password', PasswordType::class, ['toggle' => true])
+                // ...
+            ;
+        }
+
+        // ...
+    }
+
+Customizing labels and icons
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony UX TogglePassword uses English by default and svg icons from `Heroicons`_ for the toggle button.
+You can customize them by passing a ``hidden_label``, ``visible_label``, ``hidden_icon``, or a ``visible_icon`` option
+to the field (use ``null`` to disable label or icon)::
```

Comment:
```suggestion
You can customize them by passing ``hidden_label``, ``visible_label``, ``hidden_icon`` and ``visible_icon`` options
to the field (use ``null`` to disable label or icon)::
```

Reply:
Maybe specify that the labels are translated by default, and that one can pass a TranslatableMessage instance?

Also, I'm wondering if there should be a `toggle_translation_domain` when TranslatableMessage is not used for some reason (also, could help disabling translating the labels if set to `false` like it's done with the `choice_translation_domain` for the ChoiceType).